brut.apktool/apktool-lib/src/main/java/brut/androlib/res/AndrolibResources.java

```diff
@@ -46,6 +46,40 @@
 import java.util.zip.ZipOutputStream;
 
 final public class AndrolibResources {
+
+    private final Config config;
```

Comment:
Is this just a nomenclature to move them to top of file? It seems if we do this we have quite the split between files